### ISSUE TITLE
make executable

### ISSUE THREAD

**Issue by runningman84** [dhoer/chef-flywaydb#30]
Please male the /opt/flyway/flyway executable by default (chmod 755)...

**Comment by dhoer** [comment]
That sounds ok to me. Please submit the PR and make sure the rake and kitchen verify passes on travis-ci and appveyor.

**Comment by dhoer** [comment]
I released 6.0.0 with flyway now executable. Note that major bump is because I changed the install_dir default from /opt to /opt/flywaydb.

**Comment by dhoer** [comment]
Also, released with failing build because of:
```
[16:45] <dhoer> getting /opt/chefdk/embedded/lib/ruby/site_ruby/2.3.0/rubygems.rb:250:in `find_spec_for_exe': can't find gem chef-dk (>= 0.a)  using chef-current-precise
[16:45] <dhoer> did something change???
[16:45] <@coderanger> dhoer: Being fixed shortly
[16:45] <@coderanger> It's breakage from the Ruby 2.3 upgrade
[16:45] == solvip [~solvip@96-140-209-31.xdsl.hringdu.is] has joined #chef
[16:45] <@coderanger> Current will probably be broken for a day or two
```
